libparsec/crates/types/src/addr.rs

```diff
@@ -40,6 +40,7 @@ const PARSEC_ACTION_CLAIM_USER: &str = "claim_user";
 const PARSEC_ACTION_CLAIM_DEVICE: &str = "claim_device";
 const PARSEC_ACTION_CLAIM_SHAMIR_RECOVERY: &str = "claim_shamir_recovery";
 const PARSEC_ACTION_PKI_ENROLLMENT: &str = "pki_enrollment";
+const PARSEC_ACTION_ACCOUNT_EMAIL_VALIDATION: &str = "account_email_validation";
```

Comment:
nitpick:
```suggestion
const PARSEC_ACCOUNT_ACTION_EMAIL_VALIDATION: &str = "account_email_validation";
```

I would use "account" as a prefix to indicate this action is unrelated to the other ones (since we now have two types of action URLs)

In a subsequent PR, I think we should also rename `ParsceActionAddr` -> `ParsecOrganizationActionAddr` (and also `PARSEC_ACTION_XXX` -> `PARSEC_ORGANIZATION_ACTION_XXX`)